File: Trivial/Bela.java
import java.util.Scanner;

public class Bela {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = 4 * sc.nextInt();
        String b = sc.nextLine();
        int sum = 0;

        for(int i = 0; i < n; i++) {
            String card = sc.nextLine();
            String c1 = "" + card.charAt(0);
            String c2 = "" + card.charAt(1);

            if(c1.equals("A")) {
                sum += 11;
            } else if(c1.equals("K")) {
                sum += 4;
            } else if(c1.equals("Q")) {
                sum += 3;
            } else if((c1 + b).equals("J " + c2)) {
                sum += 20;
            } else if(c1.equals("J")) {
                sum += 2;
            } else if(c1.equals("T")) {
                sum += 10;
            } else if((c1 + b).equals("9 " + c2)) {
                sum += 14;
            }
        }
        sc.close();

        System.out.println(sum);
    }
}

File: solutions/bela.py
n, b = input().split()
n = 4 * int(n)
cards = []
sum = 0

while n > 0:
	cards.append(input())
	n -= 1
	
for card in cards:
	if card[0] == 'A':
		sum += 11
	elif card[0] == 'K':
		sum += 4
	elif card[0] == 'Q':
		sum += 3
	elif card[0] == 'J':
		if card[1] == b:
			sum += 20
		else:
			sum += 2
	elif card[0] == 'T':
		sum += 10
	elif card[0] == '9':
		if card[1] == b:
			sum += 14
		else:
			sum += 0
	elif card[0] == '8':
		sum += 0
	elif card[0] == '7':
		sum += 0

print(sum)
